### PR TITLE
Updated the images tags to 7.1.0 and 2.3.0

### PR DESCRIPTION
**Pull request by arodoni**

### CODE REVIEW

File: connector/replicator-with-monitoring/components-destination.yaml [hunk 1]
```diff
@@ -76,7 +76,7 @@ spec:
   tls:
     autoGeneratedCerts: true
   image:
-    application: confluentinc/cp-schema-registry:7.0.1
+    application: confluentinc/cp-schema-registry:7.1.0
     init: confluentinc/confluent-init-container:2.2.0-1
```
Comment (domenicbove) [inline connector/replicator-with-monitoring/components-destination.yaml:80]: should we switch all the init containers to 2.3?

Comment (arodoni) [reply connector/replicator-with-monitoring/components-destination.yaml:80]: I thinks so. I think I missed `2.2.0-1`s. I will search again.

File: hybrid/multi-region-clusters/README-basic.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Look at the ZK nodes.
 ```
 $ kubectl exec zookeeper-0 -it bash -n central --context mrc-central
 
-bash-4.4$ zookeeper-shell 127.0.0.1:2181
+bash-4.4$ zookeeper-shell 127.1.0.1:2181
```
Comment (domenicbove) [inline hybrid/multi-region-clusters/README-basic.md:88]: revert this

File: hybrid/multi-region-clusters/README.md [hunk 3]
```diff
@@ -432,7 +432,7 @@ Look at the ZK nodes.
 ```
 $ kubectl exec -it zookeeper-0 -n central -c zookeeper --context mrc-central -- bash
 
-bash-4.4$ zookeeper-shell 127.0.0.1:2181
+bash-4.4$ zookeeper-shell 127.1.0.1:2181
```
Comment (domenicbove) [inline hybrid/multi-region-clusters/README.md:435]: same here

**Review by domenicbove — APPROVED** [review]
LGTM, two minor changes needed

File: hybrid/multi-region-clusters/README-basic.md (hunk 2 above)
Comment (arodoni) [reply hybrid/multi-region-clusters/README-basic.md]: Donr

File: hybrid/multi-region-clusters/README.md (hunk 3 above)
Comment (arodoni) [reply hybrid/multi-region-clusters/README.md]: Done.

**Pull request merged**